EthereumKit/Networking/Request/JSONRPC.swift

```diff
@@ -80,4 +80,51 @@ public final class JSONRPC {
             return Response(id: transactionID)
         }
     }
+    
+    public struct Call: Request {
+        public typealias Response = String
+        
+        public let from: Address?
+        public let to: Address
+        public let gas: Int?
+        public let gasPrice: Int?
+        public let value: Int?
+        public let data: String?
+        public let blockParameter: BlockParameter
+        
+        public var method: String {
+            return "eth_call"
+        }
+        
+        public var parameters: Any? {
+            var txParams: [Any] = []
+            
+            if let fromAddress = from?.string {
+                txParams.append(fromAddress)
+            }
+            if let gas = gas {
+                txParams.append(gas)
+            }
+            if let gasPrice = gasPrice {
+                txParams.append(gasPrice)
+            }
+            if let value = value {
+                txParams.append(value)
+            }
+            if let data = data {
+                txParams.append(data)
+            }
+            
+            txParams.append(to.string)
+
```

Comment:
`to` must be added to parameters right after from Address

Reply:
the order should be [fromAddress?, to, gas?, gasprice?, value?, data?, blockparameter ]

Reply:
thx! After I investigate, this tx parameters should be `[String: Any]` so I'll fix it like below

```
var txParams: [String: Any] = [:]
            
if let fromAddress = from?.string {
     txParams["from"] = fromAddress
}

....

return [txParams, blockParameter.rawValue]
```